### PR TITLE
Azure Traffic Collector new service initial commit

### PR DESCRIPTION
**Pull request by kukulkarni1**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1175
    - [ ] txe markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/NetworkFunction/AzureTrafficCollector/generated/TrafficCollector.nuspec [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <version>0.1.0</version>
+    <authors></authors>
+    <owners></owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <licenseUrl></licenseUrl>
+    <projectUrl></projectUrl>
+    <description></description>
+    <releaseNotes></releaseNotes>
+    <copyright></copyright>
+    <tags></tags>
```
Comment (wyunchi-ms) [inline src/NetworkFunction/AzureTrafficCollector/generated/TrafficCollector.nuspec:14]: No need to check in this file

File: src/NetworkFunction/AzureTrafficCollector/generated/TrafficCollector.csproj [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <WarningsAsErrors />
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DelaySign>false</DelaySign>
+    <DefineConstants>TRACE;DEBUG;NETSTANDARD</DefineConstants>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
```
Comment (wyunchi-ms) [inline src/NetworkFunction/AzureTrafficCollector/generated/TrafficCollector.csproj:27]: No need to check in this file

File: src/NetworkFunction/AzureTrafficCollector/generated/TrafficCollector.format.ps1xml [hunk 3]
```diff
@@ -0,0 +1,549 @@
+<?xml version="1.0" encoding="utf-16"?>
+<Configuration>
+  <ViewDefinitions>
+    <View>
+      <Name>Microsoft.Azure.PowerShell.Cmdlets.AzureTrafficCollector.Models.AzureTrafficCollector</Name>
```
Comment (wyunchi-ms) [inline src/NetworkFunction/AzureTrafficCollector/generated/TrafficCollector.format.ps1xml:5]: No need to check in this file

File: src/NetworkFunction/AzureTrafficCollector/generated/TrafficCollector.psm1 [hunk 4]
```diff
@@ -0,0 +1,97 @@
+# region Generated 
+# ----------------------------------------------------------------------------------
+  # Load required Az.Accounts module
+  $accountsName = 'Az.Accounts'
+  $accountsModule = Get-Module -Name $accountsName
+  if(-not $accountsModule) {
+    $localAccountsPath = Join-Path $PSScriptRoot 'generated\modules'
+    if(Test-Path -Path $localAccountsPath) {
+      $localAccounts = Get-ChildItem -Path $localAccountsPath -Recurse -Include 'Az.Accounts.psd1' | Select-Object -Last 1
```
Comment (wyunchi-ms) [inline src/NetworkFunction/AzureTrafficCollector/generated/TrafficCollector.psm1:9]: No need to check in this file

**Comment by wyunchi-ms** [comment]
1. Please move the code to folder `src/TrafficCollector`. You can take the folder structure of `src/MySql` as an example.
2. Fix the missing part of `README.md`

File: src/NetworkFunction/AzureTrafficCollector/generated/TrafficCollector.format.ps1xml (hunk 3 above)
Comment (kukulkarni1) [reply src/NetworkFunction/AzureTrafficCollector/generated/TrafficCollector.format.ps1xml]: Removed this file

File: src/NetworkFunction/AzureTrafficCollector/generated/TrafficCollector.psm1 (hunk 4 above)
Comment (kukulkarni1) [reply src/NetworkFunction/AzureTrafficCollector/generated/TrafficCollector.psm1]: Removed this file

**Pull request closed without merging**